LibertyDSNP/spec
# DSNP Operations

Problem
=======
The specification lacks formal details on the operations a compliant implementation should provide.
Link to GitHub Issue(s): Closes #214 

Solution
========
This PR adds Operations tables to the Identity and Announcements pages (which are the two areas where operations have thus far been specified). For each operation, we provide a name, the principal(s) who will be invoking the operation, the inputs, and the expected outputs or state changes.

Note that this is not a spec version change, but rather a clarification of the existing spec.

Change summary:
---------------
- [ ] Updated Overview page
- [ ] Updated Identity page
- [ ] Updated Announcements page

Steps to Verify:
----------------
1. Review proposed text for accuracy and clarity.
1. Generate the website book (`npm run build`) and preview in browser to ensure there are no formatting issues.

Screenshots (optional):
-----------------------
N/A
